.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+<!--
+Please, don't forget to use the prefixes in your title, following document https://github.com/gofiber/fiber/blob/master/.github/contribuiting.md.
```

Comment:
Please, don't forget to use prefixes in your title, as described in the following document https://github.com/gofiber/fiber/blob/master/.github/contribuiting.md.